### ISSUE TITLE
Game crashes when looping is off, and other issues

### ISSUE THREAD

**Issue by jjito1** [WaterMediaTeam/watermedia#14]
logs/resources
log https://paste.ee/p/FUq27
VLC log https://paste.ee/p/dsde3
modlist: https://pixeldrain.com/u/AkFSsWit
macOS Monterey 12.5.1
chip is m1

Hello!
Recently on me and my friends modded smp, i've been crashing randomly. My friends and I were troubleshooting, and that's when I realized that when looping is off and the video ends, the game crashes. or rather, hangs. SVC (simple voice chat) still works but I can't do anything/hear anything other than SVC.

As you've said, its a rendering issue. from your words ["sending a broken buffer to opengl and causing crashes"](https://discord.com/channels/486853064284831744/1088664283262496831/1129815459689152623)

Also, some other things is twitter videos crash my game, maybe something to do with [the new twitter api](https://twitter.com/justusecobalt/status/1674690832010559488?s=20) but probably not

I wish you luck with 1.4.0/ 1.3.x ^_^ !

**Comment by bazk** [comment]
Had te same issues here, the game started hanging randomly. Only on my macOS (Ventura 13.4.1) machine, my friends on Windows machines didn't have any problems.

I was able to track it down to thiss call for `player.mediaPlayer().status().length()` on api/video/VideoLANPlayer.java:

```java
@Override
public void stopped(MediaPlayer mediaPlayer) {
    checkIfCurrentThreadHaveClassLoader();
    EV.callMediaStoppedEvent(VideoLANPlayer.this, new MediaStoppedEvent.EventData(player.mediaPlayer().status().length()));
}
```

I removed the call for .length() and replaced with a -1 as a temporary workaround:

```java
@Override
public void stopped(MediaPlayer mediaPlayer) {
    checkIfCurrentThreadHaveClassLoader();
    EV.callMediaStoppedEvent(VideoLANPlayer.this, new MediaStoppedEvent.EventData(-1));
}
```

**Comment by SrRapero720** [comment]
i hate that event handler (and i made it)

**Comment by SrRapero720** [comment]
Fixed (i think) please test it using 1.3.22
https://legacy.curseforge.com/minecraft/mc-mods/watermedia/files/4663330

**Comment by bazk** [comment]
Confirmed, issue solved for me with version 1.3.22 🎉
